### PR TITLE
new icon: unity (original, original-wordmark)

### PR DESCRIPTION
**Pull request by MichielPater**

### CODE REVIEW

**Comment by tupaschoal** [comment]
Note that this likely closes #345

File: icons/unity/unity-original.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128" xmlns:v="https://vecta.io/nano"><path d="M82.48 63.578l22.418-38.402 10.832 38.402-10.832 38.398zm-10.926 6.238l22.422 38.402-39.047-9.922-28.211-28.48zM93.969 18.93L71.555 57.34H26.719L54.93 28.855zm32 31.582L112.293.031 61.25 13.559l-7.555 13.18-15.336-.109L1 63.582l37.359 36.949h.004l15.324-.113 7.57 13.176 51.035 13.527 13.676-50.473-7.762-13.07zm0 0" fill="rgb(13.333333%,17.254902%,21.568627%)"/></svg>
```
Comment (Thomas-Boi) [inline icons/unity/unity-original.svg:1]: Just need to change the colors so it's consistent in both SVGs and we're good.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128" xmlns:v="https://vecta.io/nano"><path d="M82.48 63.578l22.418-38.402 10.832 38.402-10.832 38.398zm-10.926 6.238l22.422 38.402-39.047-9.922-28.211-28.48zM93.969 18.93L71.555 57.34H26.719L54.93 28.855zm32 31.582L112.293.031 61.25 13.559l-7.555 13.18-15.336-.109L1 63.582l37.359 36.949h.004l15.324-.113 7.57 13.176 51.035 13.527 13.676-50.473-7.762-13.07zm0 0" fill="rgb(6.666667%,4.313725%,3.529412%)"/></svg>
```

File: devicon.json [hunk 2]
```diff
@@ -3894,6 +3894,25 @@
...
+            "engine",
+            "game"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "original"
```
Comment (Thomas-Boi) [inline devicon.json:3911]: I think the `wordmark` version will look great as an icon too.
```suggestion
                "original",
                "original-wordmark"
```

**Review by Thomas-Boi — APPROVED** [review]
The SVGs and icons look great! Thank you for your work 😄. I hope you don't mind me changing the color in `original` version to match the `wordmark` version.

**Pull request merged**